langserver/format.go

```diff
@@ -28,27 +29,38 @@ func (h *LangHandler) handleTextDocumentFormatting(ctx context.Context, conn jso
 	}
 
 	filename := h.FilePath(params.TextDocument.URI)
-	bctx := h.BuildContext(ctx)
-	fset := token.NewFileSet()
-	file, err := buildutil.ParseFile(fset, bctx, nil, path.Dir(filename), path.Base(filename), parser.ParseComments)
+	unformatted, err := h.readFile(ctx, params.TextDocument.URI)
 	if err != nil {
 		return nil, err
 	}
 
-	ast.SortImports(fset, file)
+	var formatted []byte
+	if h.config.GoimportsEnabled {
+		imports.LocalPrefix = h.config.GoimportsLocalPrefix
+		var err error
+		formatted, err = imports.Process("", unformatted, nil)
```

Comment:
pass `filename` instead of `""` for the first argument. This should fix the builds.